packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -145,10 +145,10 @@ export const messages = {
     'SFDX: ISV Debugger Setup, Step 7 of 7: Converting package: %s',
   isv_debug_bootstrap_processing_package: 'Processing package: %s',
   isv_debug_bootstrap_open_project:
-    'Opening Project in new Visual Studio Code Window',
+    'Opening project in new Visual Studio Code window',
 
   error_creating_packagexml: 'Error creating package.xml: %s',
   error_extracting_org_source: 'Error extracting downloaded Apex source: %s',
   error_extracting_packages: 'Error extracting packages: %s',
-  error_updateing_sfdx_project: 'Error updating sfdx-project.json: %s'
+  error_updateing_sfdx_project: 'Error updateing sfdx-project.json: %s'
```

Comment:
@guw - The typo was actually in `error_updateing_sfdx_project`. Here's what this line should look like:

error_updating_sfdx_project: 'Error updating sfdx-project.json: %s'

Reply:
@ruthemmanuelle Thanks. Fixed. :)